src/Symfony/Component/Translation/LoggingTranslator.php

```diff
@@ -21,12 +21,9 @@
  */
 class LoggingTranslator implements TranslatorInterface, TranslatorBagInterface, LocaleAwareInterface
 {
-    /**
-     * @var TranslatorInterface|TranslatorBagInterface
-     */
-    private $translator;
+    private TranslatorInterface | TranslatorBagInterface $translator;
```

Comment:
Same here. The actual type is an intersection type, not a union type.

Reply:
fixed in 5b09467343fe027124d9e7b4f0e150defb7935a2